### PR TITLE
Create article "How bug reports are handled"

### PR DESCRIPTION
**Pull request by SimonGrn**
Create a new article about "how we handle bug reports" at PrestaShop

### CODE REVIEW

File: news/_posts/2020-07-08-how-bug-reports-are-handled.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+# Some examples
+
+## :scream: Bad bug reports
+
+- **Clearly incomplete**: you forgot to include some mandatory information like the PrestaShop version, the exact problem you encountered, or even the steps to reproduce!
+- **Wrong use of Github**: the team created a few templates to help you provide the information that’s typically needed to reproduce an issue. If you delete the whole template, we most likely won’t have this information and will lose time by asking and waiting for an answer. Same thing if you don’t write your issue in English, or just paste a 200 lines long log message with zero details.
+- **Request for support**: Bug reports should only be used to report issues which are caused by an error in the software. If the problem you report lies in your own configuration, or environment, or a module/theme that doesn’t belong to the project, the team will not be able to help you. You can use our [dedicated support](https://www.prestashop.com/fr/support) for this kind of issue!
```
Comment (SimonGrn) [inline news/_posts/2020-07-08-how-bug-reports-are-handled.md]: Maybe I should add a link to the forum and the Slack too ?

Comment (ttoine) [reply news/_posts/2020-07-08-how-bug-reports-are-handled.md]: to the slack would be good, yes.
https://github.com/PrestaShop/open-source/tree/master/slack

Comment (SimonGrn) [reply news/_posts/2020-07-08-how-bug-reports-are-handled.md]: Tell me if that's OK for you now :)

Comment (ttoine) [reply news/_posts/2020-07-08-how-bug-reports-are-handled.md]: 👍

**Pull request merged**